.github/workflows/build-images.yaml

```diff
@@ -32,12 +32,26 @@ jobs:
           echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
       - name: Set Authorino version
         id: authorino-version
+        run: |
+          echo "VERSION=latest" >> $GITHUB_ENV
+      - name: Set Authorino git sha
+        id: authorino-git-sha
         run: |
           if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
           echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
           else
-          echo "VERSION=${{ github.sha }}" >> $GITHUB_ENV
+          echo "GIT_SHA=${{ github.sha }}" >> $GITHUB_ENV
```

Comment:
Maybe we could change the name from `VERSION` to `AUTHORINO_VERSION` so this doesn't happen ? As for the `latest` tag, we could have the version set to whatever the latest tag on the repo is, `+1-dev` ?

Reply:
I'm fine with `version: latest|vX.Y.Z, commit: 1ac43d69d..., dirty: true|false`. I understand this is how other Kuadrant components are doing, right?

`version: <next>-dev` I don't like much because half of the times it will be wrong, due to `<next>` sometimes being a minor bump, sometimes a patch.

If there is one improvement I could suggest is using the name of the branch in the version, when it's not `main` (in this case we would continue injecting `latest`), nor a tag matching `vX.Y.Z` (that suggests a stable release.) IOW,

```
version: latest|vX.Y.Z|branch-name, commit: 1ac43d69d..., dirty: true|false
```

WDYT?

Reply:
Sounds good to me. Will do!